doc/sphinxman/source/optking.rst

```diff
@@ -36,12 +36,12 @@
 Geometry Optimization
 =====================
 
-.. codeauthor:: Rollin A. King
-.. sectionauthor:: Rollin A. King and Lori A. Burns
+.. codeauthor:: Rollin A. King and Alexander G. Heide
+.. sectionauthor:: Rollin A. King, Alexander G. Heide, and Lori A. Burns
 
-*Module:* :ref:`Keywords <apdx:optking>`, :ref:`PSI Variables <apdx:optking_psivar>`, :source:`OPTKING <psi4/src/psi4/optking>`
+*Module:* :ref:`Keywords <apdx:optking>`, `OPTKING <https://github.com/psi-rking/optking>`_
 
-|PSIfour| carries out molecular optimizations using a module called
+|PSIfour| carries out molecular optimizations using a python module called
```

Comment:
Should optking be in the preceding section, under

```
The following are also required for |PSIfour|, but if not detected, the
build system will automatically download and build.
```
I could remove the download from source from cmake.

Reply:
I agree it's ambiguous what section in build_planning.rst optking should go in. On one hand, optking isn't _used_ until runtime. On the other hand, since we're requiring it, it will be built at buildtime if missing. I think just choose a place for the bullet that seems reasonable. Definitely do leave the downloading of src from cmake as-is.